test/com/google/javascript/jscomp/ProcessEs6ModulesTest.java

```diff
@@ -63,7 +63,8 @@ protected int getNumRepetitions() {
     return 1;
   }
 
-  static void testModules(CompilerTestCase test, String input, String expected) {
+  static void testModules(CompilerTestCase test, String input, String expected,
+      boolean hasFileoverview) {
```

Comment:
These kinds of boolean params are annoying because you have no idea what they mean at the call site. Can we just force the caller to include the fileoverview? I know that will be somewhat annoying too but it's more explicit and clear.